mitodl/edx-sga
# fix: update pylint configurations

#### What are the relevant tickets?
None

#### What's this PR do?
Updates pylint configurations


#### How should this be manually tested?
- Check tests passing
- Check the for the over configuration changes to see if there is something that can be changed or should be changed
- There is no functionality change in the code so testing the whole edx-sga xBlock is not mandatory 

#### Where should the reviewer start?
Take a look at the failing checks to see the problems [here](https://github.com/mitodl/edx-sga/runs/7542820217?check_suite_focus=true)
